ports/nrf/boards/ARDUINO_NANO_33_BLE_SENSE/mpconfigboard.h

```diff
@@ -17,6 +17,7 @@
 #define MICROPY_PY_MACHINE_I2C          (1)
 #define MICROPY_PY_MACHINE_ADC          (1)
 #define MICROPY_PY_MACHINE_TEMP         (1)
+#define MICROPY_PY_MACHINE_UART_IRQ     (1)
```

Comment:
Is there a way to automatically set this option in `mpconfigport.h`, so it doesn't need to be done per board?  For example, all nRF52 boards get it, or use `EXTRA_FEAT`.

Reply:
It can be based on the #define of `NRF52`. But there is also a define for `MICROPY_PY_MACHINE_UART` in every board file. That could be moved to mpconfigport.h as well, especially since the uart includefile is set in mpconfigport.h.

Reply:
> But there is also a define for `MICROPY_PY_MACHINE_UART` in every board file.

Right... ideally that would be moved to `mpconfigport.h`, but maybe that can be left for another PR, to clean these up.

Do as you see fit.

Reply:
I'll leave that for a separate clean-up. There is more to re-arrange.